### PR TITLE
Jusjohns wcf cleanup

### PR DESCRIPTION
**Pull request by profexorgeek**
### Pull Request Checklist

- [ ] Rebased on top of master at time of the pull request.
- [ ] Changes adhere to coding standard in the sample.
- [ ] Consolidated NuGet packages across all projects in the sample (when changing existing package versions).
- [ ] Tested changes on the appropriate platforms (all platforms if changing the library code).

### CODE REVIEW

File: WebServices/TodoWCF/Droid/MainActivity.cs [hunk 1]
```diff
@@ -19,6 +19,12 @@ protected override void OnCreate(Bundle bundle)
             global::Xamarin.Forms.Forms.Init(this, bundle);
 
             App.Speech = new Speech();
+            
+            // NOTE: Android emulator cannot access localhost but must
+            // use a local proxy. You may need to change this if testing
+            // on a device.
+            App.SoapUrl = "http://10.0.2.2:49393/TodoService.svc";
```
Comment (davidbritch) [inline WebServices/TodoWCF/Droid/MainActivity.cs]: Can't this be specified in the shared code library? You should be able to, doing something like:

```
SoapUrl =  Device.RuntimePlatform == Device.Android ? "https://10.0.2.2:5001" : "https://localhost:5001";
```

File: WebServices/TodoWCF/TodoWCF/Constants.cs [hunk 2]
```diff
@@ -1,8 +0,0 @@
-﻿namespace TodoWCF
-{
-	public static class Constants
-	{
-		// URL of WCF service
-		public static string SoapUrl = "https://developer.xamarin.com:8081/TodoService.svc";
```
Comment (davidbritch) [inline WebServices/TodoWCF/TodoWCF/Constants.cs]: As above comment - leave the constant here, and specify the two different values for iOS/Android using Device.RuntimePlatform.

File: WebServices/TodoWCF/TodoWCF/TodoWCF.cs [hunk 3]
```diff
@@ -9,7 +9,9 @@ public class App : Application
 
 		public static ITextToSpeech Speech { get; set; }
 
-		public App ()
+        public static string SoapUrl { get; set; } = "http://localhost:49393/TodoService.svc";
```
Comment (davidbritch) [inline WebServices/TodoWCF/TodoWCF/TodoWCF.cs]: As per previous comments.

File: WebServices/TodoWCF/TodoWCF/TodoWCF.csproj [hunk 4]
```diff
@@ -1,26 +1,127 @@
...
+    <None Include="Connected Services\TodoWCFService\configuration.svcinfo" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="Connected Services\TodoWCFService\Reference.svcmap">
+      <Generator>WCF Proxy Generator</Generator>
+      <LastGenOutput>Reference.cs</LastGenOutput>
+    </None>
+  </ItemGroup>
+  <ItemGroup>
+    <Reference Include="Xamarin.Forms.Core, Version=2.0.0.0, Culture=neutral, processorArchitecture=MSIL">
```
Comment (davidbritch) [inline WebServices/TodoWCF/TodoWCF/TodoWCF.csproj]: So you've replaced the .NET Standard Lib with a PCL lib? I understand why. I've just got to think about how this impacts future mass updates of the repo.

File: WebServices/TodoWCF/TodoWCF/packages.config [hunk 5]
```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Xamarin.Forms" version="3.6.0.264807" targetFramework="portable46-net451+win81" />
```
Comment (davidbritch) [inline WebServices/TodoWCF/TodoWCF/packages.config]: Can you use 3.6.0.220655 instead? All samples in the repo are on that version, which makes it easy to do a find/replace for future mass updates.

File: WebServices/TodoWCF/TodoWCF/TodoWCF.csproj [hunk 6]
```diff
@@ -1,26 +1,127 @@
...
+      <HintPath>..\packages\Xamarin.Forms.3.6.0.264807\lib\netstandard1.0\Xamarin.Forms.Core.dll</HintPath>
```
Comment (davidbritch) [inline WebServices/TodoWCF/TodoWCF/TodoWCF.csproj]: Can you use 3.6.0.220655 instead? All samples in the repo are on that version, which makes it easy to do a find/replace for future mass updates.

File: WebServices/TodoWCF/UWP/TodoWCF.UWP.csproj [hunk 7]
```diff
@@ -88,10 +88,15 @@
     <UseDotNetNativeToolchain>true</UseDotNetNativeToolchain>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.NETCore.UniversalWindowsPlatform" Version="6.0.12" />
-    <PackageReference Include="Xamarin.Forms" Version="3.6.0.220655" />
+    <PackageReference Include="Microsoft.NETCore.UniversalWindowsPlatform" Version="6.2.8" />
+    <PackageReference Include="Xamarin.Forms" Version="3.6.0.264807" />
```
Comment (davidbritch) [inline WebServices/TodoWCF/UWP/TodoWCF.UWP.csproj]: Can you use 3.6.0.220655 instead? All samples in the repo are on that version, which makes it easy to do a find/replace for future mass updates.

File: WebServices/TodoWCF/TodoWCF/TodoWCF.csproj (hunk 4 above)
Comment (profexorgeek) [reply WebServices/TodoWCF/TodoWCF/TodoWCF.csproj]: Honestly I feel like this WCF project should probably be excluded from auto updates as it is fairly fragile. As part of this cleanup I ended up fixing issues caused by automatic updates to VS version, forms version and .NET Core!

File: WebServices/TodoWCF/UWP/TodoWCF.UWP.csproj (hunk 7 above)
Comment (profexorgeek) [reply WebServices/TodoWCF/UWP/TodoWCF.UWP.csproj]: I did downgrade this, however it resulted in errors (which is why I upgraded in the first place). After downgrade the UWP project threw this unhandled exception while trying to handle image rendering: System.TypeLoadException: Could not find Windows Runtime type 'Microsoft.Graphics.Canvas.UI.Xaml.CanvasImageSource'

This issue is discussed here: https://github.com/xamarin/Xamarin.Forms/issues/5188

The workaround required adding the Win2D.uwp package and upgrading the target framework version in the UWP project.

**Pull request merged**